README.md

```diff
@@ -81,7 +81,7 @@ For more information, see the [wiki](https://github.com/Azure/azure-storage-fuse
     * [OPTIONAL] **--streaming=false** : Enable read streaming of files instead of disk-caching. This option works only with read-only mount. This option is only available from version 1.4.0
     * [OPTIONAL] **--stream-cache-mb=500** : Limit total amount of data being cached in memory to conserve memory footprint of blobfuse.
     * [OPTIONAL] **--max-blocks-per-file=3** : Maximum number of blocks to be cached in memory for a read streaming.
-    * [OPTIONAL] **--block-size-mb=16** : Size (in MB) of a block to be downloaded during streaming.
+    * [OPTIONAL] **--block-size-mb=16** : Size (in MB) of a block to be downloaded during streaming. If configured block-size is <= 64MB then block will be downloaded in a single thread. For higher block size parts of it will be downloaded in parallel. "--max-concurrency" parameter can be used to control parallelism factor. When higher block size is configured, memory usage of blobfuse will be high as these blocks are cached in memory only.
```

Comment:
should we hardcode this 64mb value? why not make this user settable as well?

Reply:
Just felt whether to use single thread or parallel download shall be left to internal implementation. Some one may configure 1mb block also be to downloaded in parallel thread, which will be overkill.